VirtoCommerce.Storefront.IntegrationTests/Cart/ApiCartControllerTests.cs

```diff
@@ -112,50 +155,55 @@ public async Task GetCartItemsCount_IfCartHasNoItems_ShouldReturnZero()
         public async Task GetCartItemsCount_IfCartHasItems_ShouldReturnExactItemsCount()
         {
             //arrange
-            var quantity = (new Fixture()).Create<int>();
+            var quantity = new Fixture().Create<int>();
 
             _client
                 .Login("admin", "store")
-                .СlearCart()
-                .InsertCartItem(new AddCartItem { Id = "9cbd8f316e254a679ba34a900fccb076", Quantity = quantity });
+                .ClearCart()
+                .InsertCartItem(new AddCartItem { Id = Product.Quadcopter, Quantity = quantity });
 
             //act
             var result = await _client.GetAsync<int>(TestEnvironment.CartItemsCountEndpoint);
 
             //assert
             result.Should().Be(quantity);
-            _client.СlearCart().Logout();
+
+            _client.ClearCart().Logout();
         }
 
         [Fact]
         public async Task ClearCart_IfCartExists_ShouldReturnEmptyCart()
         {
             //arrange
             _client
-                .Login("admin", "store")
-                .СlearCart();
+                .Login("admin", "store");
+
+            await RemoveAllCoupons();
+
+            _client
+                .ClearCart();
 
             //act
-            var result = await GetCart(_client);
+            var result = await _client.GetCart();
 
             //assert
-            GetCartComparationResult(result, "GetEmptyCartForAdmin")
+            GetCartComparationResult(result, "GetEmptyCartForAdmin", new[] { "$" }, new[] { "id" })
```

Comment:
Expected string to be <null>, but found "{
      "isAnonymous": [
        true,
        false
      ],
      "customerName": [
        "Anonymous",
        "admin"
      ],
      "type": [
        "cart",
        0,
        0
      ]
    }